project_event/models/project_task.py

```diff
@@ -216,6 +217,34 @@ def check_numeric(self):
         except Exception as exception:
             raise ValidationError(_('The Input value should be numeric'))
 
+    real_date_start = fields.Datetime(
+        string='Actual Start Time',
+        default=None,
```

Comment:
@jananjoy what about default? by default it should be the same date as date_start